src/vm/methodtablebuilder.cpp

```diff
@@ -10257,6 +10278,16 @@ void MethodTableBuilder::CheckForSystemTypes()
         {
             pMT->SetIsNullable();
         }
+#ifdef FEATURE_SPAN_OF_T
+        else if (strcmp(name, g_SpanName) == 0)
+        {
+            pMT->SetIsByRefLike();
+        }
+        else if (strcmp(name, g_ReadOnlySpanName) == 0)
+        {
+            pMT->SetIsByRefLike();
+        }
```

Comment:
OK, let's leave them here as-is. We want to get quickly to a point where we understand where we are with perf. We can tweak the APIs in both places and sync them periodically.